docs/user/debugging/index.md

```diff
@@ -12,12 +12,25 @@
 
 There are 3 available commands that you can use with the Glean SDK debug tools
 
-- `logPings`: This is either true or false and will cause pings that are submitted to also be echoed to the device's log
+- `logPings`: This is either true or false and will cause pings that are submitted to also be echoed to the device's log.
 - `tagPings`: This command will tag outgoing pings with the provided value, in order to identify them in the Glean Debug View. Tags need to be string with upper and lower case letters, numbers and dashes, with a max length of 20 characters.
 - `sendPing`: This command expects a string name of a ping to force immediate collection and submission of.
 
 Different platforms have different ways to send these commands.
 
+### Enabling debugging features through environment variables
+
+Some of the debugging features described above may also be enabled using environment variables:
+
+- `logPings`: May be set by the `GLEAN_LOG_PINGS` environment variable. The accepted values are
+`true` or `false`. Any other value will be ignored.
+- `tagPings`: May be set by the `GLEAN_DEBUG_VIEW_TAG` environment variable. Any valid HTTP header value maybe set here
+(e.g. any value that matches the regex `[a-zA-Z0-9-]{1,20}`). Invalid values will be ignored.
+
+These variables must be set at runtime, not at compile time. They will be checked upon Glean initialization.
+
+Enabling debugging features using environment variables is available for all supported platforms.
```

Comment:
Except Android :)

Reply:
Hm, but it is available for Android. We just haven't figured out how to set environment variables for Android apps without changing the binary, if the user doesn't care about changing the binary they can totally use this method.

Reply:
Mh, ok, you have a point hee.